contracts/src/bridge/Outbox.sol

```diff
@@ -29,8 +29,21 @@ contract Outbox is DelegateCallAware, IOutbox {
     L2ToL1Context internal context;
     uint128 public constant OUTBOX_VERSION = 2;
 
+    modifier outboxIsRunning() {
+        if(context.l2Block == type(uint128).max) revert OutboxNotStart();
+        _;
+    }
+
     function initialize(address _rollup, IBridge _bridge) external onlyDelegated {
         if (rollup != address(0)) revert AlreadyInit();
+        bytes32 initialBytes = keccak256(abi.encode(block.timestamp));
+        context = L2ToL1Context({
+            l2Block: type(uint128).max,
+            l1Block: type(uint128).max,
+            timestamp: type(uint128).max,
+            outputId: initialBytes,
+            sender: address(this)
+        });
```

Comment:
instead of hardcoding these values here, I would make a struct that is the `EMPTY_CONTEXT` and set it here
You can't create a `constant` struct but can use the workaround here https://github.com/ethereum/solidity/issues/4037 which should make it cheap

Reply:
this is useful because then in the other view functions like `l2ToL1Sender` you can check if we have the `EMPTY_CONTEXT` value

Reply:
also I'd use different values for outputId and sender
```suggestion
        context = L2ToL1Context({
            l2Block: type(uint128).max,
            l1Block: type(uint128).max,
            timestamp: type(uint128).max,
            outputId: bytes32(type(uint256).max),
            sender: address(type(uint160).max)
        });
```